docs/OPTIONS.md

```diff
@@ -40,20 +41,44 @@ remote_compression_level={LEVEL}
  
 ## Ignore Rule Files
 
+To speedup execution you might want to **ignore large directories and files** not required for remote command such as `/.git` and so on. 
+
+Please see our samples:
```

Comment:
Refer to `README.md` section instead. You’ll have to copy this every time from there otherwise.